platforms/scripts/cmake_android_arm.sh

```diff
@@ -4,4 +4,4 @@ cd `dirname $0`/..
 mkdir -p build_android_arm
 cd build_android_arm
 
-cmake -DCMAKE_BUILD_WITH_INSTALL_RPATH=ON -DCMAKE_TOOLCHAIN_FILE=../android/android.toolchain.cmake $@ ../..
+cmake -DANDROID_NATIVE_API_LEVEL=21 -DANDROID_ABI="armeabi-v7a-hard with NEON" -DCMAKE_BUILD_WITH_INSTALL_RPATH=ON -DCMAKE_TOOLCHAIN_FILE=../android/android.toolchain.cmake $@ ../..
```

Comment:
@powderluv, please revert this script, it could be used by someone who does not expect this change. You can create new script for building `hard` version.